test-data/unit/lib-stub/enum.pyi

```diff
@@ -43,3 +43,8 @@ class IntFlag(int, Flag):
 
 class auto(IntFlag):
     value: Any
+
+
+# It is python-3.11+ only:
+class StrEnum(str, Enum):
+    def __new__(cls: Type[_T], value: int | _T) -> _T: ...
```

Comment:
[Over at typeshed](https://github.com/python/typeshed/pull/6580) I corrected this to 

```python
    def __new__(cls: Type[_T], value: str | _T) -> _T: ...
```

Reply:
Yes, sure. This is a typo. Thanks!

Reply:
No problem!